DevOpsatMicrosoft.md

```diff
@@ -11,7 +11,32 @@ Deploy: We have automated deployments and ship every 3 weeks with hotfixes deliv
 Measure: Follow build, measure and learn principle and this is incorporated in everything we do from planning to delivery to customers.
 
 # What are the main learnings from your DevOps journey?
-I would say the first and key one is DevOps is a journey, you are never done and can keep improving to deliver value to your customer. Secondly, culture plays a key part, the culture of growth mindset, the culture of experiment and this learning is key in making your DevOps journey successful. It is a transformation from celebrating success to celebrating failure and learning from those mistakes and course correct to add value to customers.
+We have been on this journey for a decade now and we aren't done yet. However at various stages through this journey we have had numerous learnings. While it's difficult to list all of them here, we have listed a few that were critical to our success. 
+
+## Break long release cycles 
```

Comment:
We can talk about the following topics as well:
1) Agile
2) Culture / Growth Mindset
3) Move to git

Also, we should keep the tone more of "what has been our journey" than "prescriptive"? E.g. We can avoid saying - "you break your long monolithic release cycles", we can say - We had a problem therefore we broke our release cycles .. which is already coming up in the next line..

Reply:
Makes sense, let me change the tone. For the additional topics, I have intentionally not included some of these as I wanted it to be more tangible for the consumer of this article at the same time having some novelty. We can have a quick chat if you still feel we should.